### PR TITLE
Remove all references to flaming fir (#21)

### LINKED ISSUE #21: Remove flaming fir references
Plug node is inoperable with the flaming fir testnet so all references should be removed.

e.g. https://github.com/plugblockchain/plug-blockchain/blob/cfc1eaac64b637d2884da91c49d34e8940c8e993/node/cli/src/lib.rs#L65

### PR DESCRIPTION
**Pull request by aliXsed**
Closes https://github.com/plugblockchain/plug-blockchain/issues/21

### CODE REVIEW

File: node/cli/src/lib.rs [hunk 1]
```diff
@@ -53,16 +53,13 @@ pub enum ChainSpec {
 	/// Whatever the current runtime is, with simple Alice/Bob auths.
 	LocalTestnet,
 	/// The Flaming Fir testnet.
```
Comment (jordy25519) [inline node/cli/src/lib.rs]: Should remove this comment and keep the one below

**Review by jordy25519 — APPROVED** [review]
LGTM! assuming CI passes

Can change comment to say "Closes #21" and github automation will close the issue on merge

File: node/cli/src/lib.rs (hunk 1 above)
Comment (aliXsed) [reply node/cli/src/lib.rs]: Will do

**Review by cowboy-bebug — APPROVED** [review]
Yep looks good!

**Pull request merged**